smithy-kotlin-codegen/src/test/kotlin/software/amazon/smithy/kotlin/codegen/core/KotlinDelegatorTest.kt

```diff
@@ -73,4 +79,55 @@ class KotlinDelegatorTest {
         // ensure symbol wasn't imported as an alias by default
         contents.shouldNotContain("as BigInteger")
     }
+
+    @Test
+    fun itRendersGeneratedDependencies() {
```

Comment:
nit/suggestion

maybe worth a test to verify that context key/value pairs can be used from within symbol renderers